### ISSUE TITLE
Error remove already removed user 

### ISSUE THREAD

**Issue by git-no** [fnichol/chef-user#54]
```
user_account 'xxx' do
  action  :remove
end

brings error in knife solo cook

```

Error executing action `remove` on resource 'user_account[xxx]'

```
```

**Comment by CpuID** [comment]
Getting the same issue (same getpwnam call failing) when using chef-client with -W (whyrun/simulate).

**Comment by theckman** [comment]
This should be fixed in the `0.5.0` release of this cookbook. If you are still seeing the problem please raise a new issue and I'd be happy to take a look.
